packages/web-components/fast-foundation/src/calendar/calendar.ts

```diff
@@ -92,6 +92,13 @@ export class FASTCalendar extends FASTElement {
         this.dateFormatter.locale = this.locale;
     }
 
+    /**
+     * Weekday that the calendar should start on, defaults to Sunday
+     * @public
+     */
+    @attr({ attribute: "first-day" })
```

Comment:
I think you'll want to add a convert to this, otherwise setting the attr could be assigned a string value instead of a number: https://www.fast.design/docs/api/fast-element.nullablenumberconverter

Reply:
Good catch. I missed that.